hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -447,11 +448,17 @@ public static DataStreamSink<Object> dummySink(DataStream<Object> dataStream) {
   }
 
   public static String opName(String operatorN, Configuration conf) {
-    return operatorN + ": " + conf.getString(FlinkOptions.TABLE_NAME);
+    return operatorN + ": " + getDbTableName(conf);
   }
 
   public static String opUID(String operatorN, Configuration conf) {
-    return "uid_" + operatorN + "_" + conf.getString(FlinkOptions.TABLE_NAME);
+    return "uid_" + operatorN + "_" + getDbTableName(conf);
+  }
+
+  public static String getDbTableName(Configuration conf) {
+    String databaseName = conf.getString(FlinkOptions.DATABASE_NAME);
+    return StringUtils.isNullOrEmpty(databaseName) ? conf.getString(FlinkOptions.TABLE_NAME)
+        : databaseName + "#" + conf.getString(FlinkOptions.TABLE_NAME);
```

Comment:
Why using `#` as the separator?

Reply:
Yes we can change to use  `.`  as seperator to since it's the convention.